test/Interop/C/bounds-safety/Inputs/bounds-attributed-function.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <ptrcheck.h>
```

Comment:
Is this header file guaranteed to be on every platform we deploy to?

Reply:
ah probably not, no

Reply:
or actually, maybe? We ship it with clang, and we ship clang with swift. So I think it ought to be there